src/components/card/CartCard.jsx

```diff
@@ -68,18 +81,18 @@ const CartCard = (product) => {
           </button>
         </div>
         <p className={classes['prod-price']}>₹{price}</p>
-        <span className={classes.fav}>
+        <span onClick={wishlistClickHandler} className={classes.fav}>
```

Comment:
You could use one type of convention for using classes. Like the one above using [...] would be better. The point being just use any one kind.
E.g.
`className = {classes['fav']}`

Reply:
All right ill keep it in mind, thanks.